spotifycli/spotifycli.py

```diff
@@ -212,45 +246,63 @@ def show_art_url():
 
 
 def get_spotify_property(spotify_property):
-    try:
-        dbus_addr = DBusAddress(
-            bus_name="org.freedesktop.DBus",
-            object_path="/org/freedesktop/DBus",
-            interface="org.freedesktop.DBus",
+    dbus_addr = DBusAddress(
+        bus_name="org.freedesktop.DBus",
+        object_path="/org/freedesktop/DBus",
+        interface="org.freedesktop.DBus",
+    )
+    connection = open_dbus_connection(bus="SESSION")
+
+    list_names_call = new_method_call(
+        remote_obj=dbus_addr, method="ListNames", signature=""
+    )
+    reply = connection.send_and_get_reply(list_names_call)
+    if reply.header.message_type.name == 'error':
+        raise SpotifyCLIException(
+            "Could not retrieve list of services (bus names) from dbus." +
+            f"Full error text:\n{reply.header.message}"
```

Comment:
Another "known" error condition (although very unlikely), where dbus does not respond.  If this happened with current code, the "names" variable in the next line would have invalid data and end up with a strange Python failure a few lines down.  With this code, the user will see the failure is because of this dbus problem, and could be helpful in identifying the issue right away.